src/app/authentication/resources/application.yml

```diff
@@ -1 +1,6 @@
-package resources
\ No newline at end of file
+database:
+  DBUsername: "root"
+  DBPassword: "S@nket123"
+  DBName: "customer_database"
+  DBHost: "localhost"
+  DBPort: "3306"
```

Comment:
keep the keys in small case as per the convention.

Reply:
Done